src/coreclr/md/compiler/mdutil.h

```diff
@@ -80,4 +80,7 @@ class LOADEDMODULES : public CDynArray<RegMeta *>
 
 #endif //FEATURE_METADATA_IN_VM
 
+#define START_MD_PERF()
+#define STOP_MD_PERF(MDTag)
```

Comment:
@jkotas, I found some time to follow up on https://github.com/dotnet/runtime/pull/42782#discussion_r495528763 :)

I can clean up `START/STOP_MD_PERF` calls if we don't need this instrumentation for anything.

Reply:
Cleaning up `START/STOP_MD_PERF` sounds good to me. Thanks!